index.markdown

```diff
@@ -93,6 +93,10 @@ carousel_images:
     title: FreeSurfer support
     description: Import volumes, segmentations, surfaces from FreeSurfer then edit and analyze them. <a href="https://youtu.be/v0rGbno4h2M?t=27"> video > </a> <a href="https://discourse.slicer.org/t/12751"> learn more > </a>
     button_text: learn more
+  - image: assets/img/image-carousel/SlicerDownloads.png
+    title: 1 million downloads
+    description: 3D Slicer reaches 1 million downloads worldwide. <a href="https://download.slicer.org/download-stats/"> learn more > </a>
```

Comment:
```suggestion
    description: 3D Slicer reaches 1 million downloads worldwide. <a href="https://www.slicer.org/wiki/Main_Page/SlicerCommunity"> learn more > </a>
```

Instead of directly referencing the download stats I suggest to link the community page where all the other stats are also listed.

Reply:
Sounds good

Reply:
<img width="561" alt="SlicerDownloads" src="https://user-images.githubusercontent.com/1847492/160609701-4084d336-f6f0-4fa9-8bc6-5b04509d72ab.png">

Reply:
Thanks for the screenshot, I updated the link and included the image :rocket:

Reply:
Here is a cropped version of the screenshot
<img width="561" alt="SlicerDownloads" src="https://user-images.githubusercontent.com/1847492/160636281-238c2005-db1c-4c37-83a7-166d9dbe0017.png">